packages/server/src/api/controllers/deploy/index.js

```diff
@@ -31,13 +35,40 @@ async function replicateCouch({ instanceId, clientId, credentials }) {
   await Promise.all(replications)
 }
 
+async function getCurrentInstanceQuota(instanceId) {
+  const db = new PouchDB(instanceId)
+  const records = await db.allDocs({
+    startkey: "record:",
+    endkey: `record:\ufff0`,
+  })
+  const existingRecords = records.rows.length
+
+  const designDoc = await db.get("_design/database")
+
+  return {
+    records: existingRecords,
```

Comment:
Just a question, in terms of local storage and users created, do we want to count them as well?

Reply:
Yep, we do for users, but the storage will be incremented when we upload things based on your PR, right?

Reply:
true yes, sorry I didn't understand that the `upload` endpoint would be used for when the storage is send up, that makes sense in that case!

Reply:
Updated for users 👍